.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run gulp lint release
+    - run: yarn install --frozen-lockfile
```

Comment:
What does that mean? I noticed you remove the package lock file. Shouldn't this be tied to a specific version of terriajs?

Reply:
Ah, now I see there is a yarn.lock added instead. I guess that replaces it and is what is used here?